.github/workflows/cff-validator.yml

```diff
@@ -0,0 +1,20 @@
+on:
+  push:
+    paths:
+      - CITATION.cff
+  workflow_dispatch:
+
+name: CITATION.cff
+jobs:
+  Validate-CITATION-cff:
+    runs-on: ubuntu-latest
+    name: Validate CITATION.cff
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why does this need a secret?

Reply:
Maybe I don't understand how workflows work&mdash;isn't a key/token always required in order to run/access the environment?

Moot point in any case if we're not going to add this workflow. 👍